frontend/projects/upgrade/src/environments/environment.ts

```diff
@@ -35,8 +35,8 @@ export const environment = {
     loginUser: `${endpointApi}/login/user`, // Used to create a new user after login if doesn't exist in DB
     getAllUsers: `${endpointApi}/users/paginated`,
     userRole: `${endpointApi}/users/role`,
-    excludeUsers: `${endpointApi}/exclude/user`,
-    excludeGroups: `${endpointApi}/exclude/group`,
+    excludeUsers: `${endpointApi}/explicitExclude/global/user`,
```

Comment:
@RidhamShah can you also update the API changes in other environment files also. Please keep in mind that any change in the `environment.ts` file should also be reflected in other files also.

Reply:
@RidhamShah can you provide update on this?

Reply:
Code updated